### ISSUE TITLE
unable to merge my fork from upstream/master

### ISSUE THREAD

**Issue by vincent3569** [desktop/desktop#4130]
### Description

I am unable to merge my fork from upstream/master.
https://github.com/vincent3569/zenphoto-perso is a fork from https://github.com/zenphoto/zenphoto.

I want to keep my fork sync with upstream (my master branch is 1 commit behind zenphoto:master (as you can show here: https://github.com/vincent3569/zenphoto-perso/tree/master)
Before that, I follow this help: https://help.github.com/articles/configuring-a-remote-for-a-fork/

Then, I try to merge upstream/master into my fork (menu branch/merge into current branch).
But, when I select upstream/master to merge from, desktop tells me there is nothing to merge.
And of course there a new commit into upstream/master.

fyi, if I follow this help: https://help.github.com/articles/syncing-a-fork/, this syncs my fork with upstream (see this commit as example: https://github.com/vincent3569/zenphoto-perso/commit/7376954868437bc138d155db799eb259ea091f41)

![capture](https://user-images.githubusercontent.com/1224809/36788978-4aa145a6-1c90-11e8-9dea-9cc112ae9c5d.PNG)

### Version

**GitHub Desktop version:** [1.1.1-beta1]

**OS version:** [Microsoft Windows 10.0.16299.125]

### Logs

[2018-02-28.desktop.beta.log](https://github.com/desktop/desktop/files/1767090/2018-02-28.desktop.beta.log)

**Comment by shiftkey** [comment]
@vincent3569 apologies for the delay in following up with this.

I had a look at the current state of your fork's `master` and it's both ahead and behind the upstream's `master` branch:

<img width="1002" src="https://user-images.githubusercontent.com/359239/39560233-a73aab46-4edf-11e8-93d0-093d8f32d03a.png">

I'm testing this against `1.1.2-beta4` which also contains our new Compare tab, and this showed the same compare result:

![](https://user-images.githubusercontent.com/359239/39560325-79178576-4ee0-11e8-949e-2d76134d60b8.gif)

This should make keeping your fork up to date - the merge experience becomes more interactive and better aligns with what GitHub The Website is displaying.

I also got the same number of commits in the merge dialog:

<img width="524" src="https://user-images.githubusercontent.com/359239/39560359-bcd94f9c-4ee0-11e8-864f-de901d706662.png">

The best theory I've got around what's changed is that maybe your local repository hadn't fetched `upstream` recently, meaning it wasn't aware of the new commit.

Let me know if you're still encountering this problem, and maybe I can try some other testing.
